src/main/resources/data/computercraft/lua/bios.lua

```diff
@@ -937,6 +947,11 @@ settings.define("bios.strict_globals", {
     description = "Prevents assigning variables into a program's environment. Make sure you use the local keyword or assign to _G explicitly.",
     type = "boolean",
 })
+settings.define("shell.autocomplete_hidden", {
+    default = false,
+    description = [[Autocomplete hidden files and folders (those starting with ".")]],
```

Comment:
```suggestion
    description = [[Autocomplete hidden files and folders (those starting with ".").]],
```

Sorry, tiny nitpick!

Reply:
In my defense, the list.show_hidden settings doesn't have a finishing dot, I copied from that ;P

But sure, I can change it ;)

Reply:
Oh, does it not? Can you fix that one as well then. That's probably my fault :flushed:, I'm always forgetting to end sentences.

Reply:
Haha, I can check that and fix it in all the settings descriptions when I'm able to have another look at the code :smile:

Reply:
Done, pushing it now, only list.show_hidden missed the .